src/control/server/config/server.go

```diff
@@ -553,6 +553,8 @@ func (cfg *Server) validateMultiServerConfig(log logging.Logger) error {
 	seenValues := make(map[string]int)
 	seenScmSet := make(map[string]int)
 	seenBdevSet := make(map[string]int)
+	seenIdx := 0
+	var seenBdevCount, seenTargetCount, seenHelperStreamCount *int
```

Comment:
Wouldn't block on it, but since these are signed ints, maybe we could init them with negative values as a flag, instead of using pointers?